File: lib/models/prediccion_model.dart
class PrediccionModel {
  final String id;
  final String partidoId;
  final String userId;
  final String userName;
  final int golesLocal;
  final int golesVisitante;
  final DateTime fecha;

  PrediccionModel({
    required this.id,
    required this.partidoId,
    required this.userId,
    required this.userName,
    required this.golesLocal,
    required this.golesVisitante,
    required this.fecha,
  });
}

File: app/src/main/java/com/example/hockey_app/data/models/PrediccionModel.kt
package com.example.hockey_app.data.models

import kotlinx.serialization.Serializable

@Serializable
data class PrediccionModel(
    val id: String? = null,
    val partido_id: String,
    val user_id: String,
    val user_name: String,
    val goles_local: Int,
    val goles_visitante: Int,
    val fecha: String? = null
)
